tests/core/test_package_cache_data.py

```diff
@@ -456,15 +457,27 @@ def test_cover_get_entry_to_link():
 
     with make_temp_package_cache() as pkgs_dir:
         first_writable = PackageCacheData(pkgs_dir)
+        assert first_writable._package_cache_records is not None
         first_writable._package_cache_records[exists] = exists
         PackageCacheData.get_entry_to_link(exists_record)
         del first_writable._package_cache_records[exists]
 
 
 def test_cover_fetch_not_exists():
-    with pytest.raises(CondaHTTPError):
+    """
+    Conda collects all exceptions raised during ProgressiveFetchExtract into a
+    CondaMultiError. TODO: Is this necessary?
+    """
+    with pytest.raises(CondaMultiError):
         ProgressiveFetchExtract(
-            [MatchSpec(url="http://localhost:8080/conda-test/fakepackage-1.2.12-testing_3.conda")]
+            [
+                MatchSpec(
+                    url="http://localhost:8080/conda-test/fakepackage-1.2.12-testing_3.conda"
+                ),
+                MatchSpec(
+                    url="http://localhost:8080/conda-test/phonypackage-0.0.1-testing_3.conda"
+                ),
+            ]
```

Comment:
I didn't go so far as to count the errors in CondaMultiError. Should vary based on thread count?